molecule/default/roles/test_glauth/files/glauth-expected.yaml

```diff
@@ -34,6 +34,7 @@ users:
   - name: testuser
     uidnumber: 1001
     primaryGroup: 501  # TODO: lowercase?
+    # yamllint disable-line rule:quoted-strings
     passbcrypt: "243262243035242e4e6c53586d49344d5148666741576e4a6d57376e6539\
```

Comment:
https://github.com/adrienverge/yamllint/issues/663 seems the relevant issue here

Reply:
I filed https://github.com/adrienverge/yamllint/issues/753 with what I know